pkg/reconciler/v1alpha1/service/service_test.go

```diff
@@ -276,6 +277,20 @@ func TestReconcile(t *testing.T) {
 		}, {
 			Object: route("update-route-and-config", "foo", WithRunLatestRollout),
 		}},
+	}, {
+		Name: "runLatest - update route and service labels",
```

Comment:
probably meant to be `"runLatest - update route and config labels"`

Reply:
Good catch!  Done.